### PR TITLE
Support more linux paths and shorter rect64 values. Ignore multiple backuphash values. 

### PR DESCRIPTION
**Pull request by athwds**

### CODE REVIEW

File: digikam_db.py [hunk 1]
```diff
@@ -50,14 +50,21 @@ def volume_uuid_to_mountpoints(uuid: str) -> Set[str]:
...
-                self.album_roots[Path(mountpoint) / specific_path] = id
+            logging.info('id=%s specific_path=%s identifier=%s' % (id, specific_path, identifier))
+            if identifier.startswith('volumeid:?uuid='):
+                if specific_path.startswith('/'):
+                    specific_path = specific_path[1:]
+                for mountpoint in volume_uuid_to_mountpoints(identifier[15:]):
+                    self.album_roots[Path(mountpoint) / specific_path] = id
+            elif identifier.startswith('volumeid:?path='):
+                self.album_roots[identifier[15:]] = id
+            else:
```
Comment (Philipp91) [inline digikam_db.py]: Please make this also elif, and then
```
else:
  raise ValueError('Unsupported ...' % identifier)
```

File: rect64.py [hunk 2]
```diff
@@ -6,7 +6,8 @@ def parse_rect64(data: str) -> Tuple[float, float, float, float]:  # left, top,
     # Strip the rect64() from the outside.
     assert data.startswith("rect64(") and data.endswith(")"), input
     data = data[7:-1]
-    assert len(data) >= 8
+    logging.info ("data=",data)
```
Comment (Philipp91) [inline rect64.py]: Please no space before `(`, but a space after `,`.

File: digikam_db.py [hunk 3]
```diff
@@ -24,7 +24,7 @@ class DigikamDb(object):
     def __init__(self, file: Path):
         self.file = file
         self.conn = sqlite3.connect(file)
-
+        logging.info("file=",file)
```
Comment (Philipp91) [inline digikam_db.py]: Please add a space after the `,`

**Review by Philipp91 — CHANGES_REQUESTED** [review]
Thank you! I can't test this myself but I trust you that it works. I only have some style-related comments.

Also, all files seem to be touched with `100644 → 100755`, i.e. making them executable. Do you know why this happened? Can you somehow undo it? (If not, I will do so before merging.)
If you use Windows, the Internet recommends setting the `core.filemode` option to false to prevent this in future commits.

**Comment by athwds** [comment]
i will clean up code and create a new pull request

**Pull request closed without merging**